app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIClient.java

```diff
@@ -339,12 +340,20 @@ public void onFailure(Call<State> call, Throwable t) {
     public void postImg(final Context context, final ProductImage image) {
      /**  final LoadToast lt = new LoadToast(context);
         lt.show();**/
-
+        Dialog dialog;
+        MaterialDialog.Builder builder = new MaterialDialog.Builder(context)
```

Comment:
No need for builder variable since it's not used elsewhere, just use dialog.

Reply:
Yeah, I did that changes and in the next commit @brandenfung you can see that.